webservices/resources/legal.py

```diff
@@ -133,17 +133,19 @@ def get(self, q='', from_hit=0, hits_returned=20, type='all', **kwargs):
         results['total_all'] = total_count
         return results
 
-def apply_mur_specific_query_params(query, **kwargs):
+def apply_mur_specific_query_params(query, q='', **kwargs):
     if kwargs.get('mur_no'):
         query = query.query('terms', no=kwargs.get('mur_no'))
     if kwargs.get('mur_respondents'):
         query = query.query('match', respondents=kwargs.get('mur_respondents'))
+    if kwargs.get('mur_dispositions'):
+        query = query.query('match', disposition__data__disposition=kwargs.get('mur_dispositions'))
     if kwargs.get('mur_election_cycles'):
         query = query.query('term', election_cycles=kwargs.get('mur_election_cycles'))
-    if kwargs.get('mur_document_category') and kwargs.get('mur_document_text'):
+    if kwargs.get('mur_document_category'):
         combined_query = [
             Q('match', documents__category=kwargs.get('mur_document_category')),
-            Q('match', documents__text=kwargs.get('mur_document_text'))]
+            Q('match', documents__text=q)]
```

Comment:
oh great! Then you just need to change the args to accept a list and change the query to be a `terms` instead of `match`.